Measurement/twinkles_light_curves.ipynb

```diff
@@ -0,0 +1,480 @@
+    "plt.xlabel('X Pixels', size=28)\n",
+    "plt.ylabel('Y Pixels', size=28)\n",
+    "plt.xticks(size=18)\n",
+    "plt.yticks(size=18)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We've now loaded an image but need to translate the ra, dec coordinates of the truth catalog into pixel values in the image. To do this we will get the WCS information for the `calexp`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wcs = calexp.getWcs()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we will pick a twinkles system to look at. We will use twinkles system 27 since it has a good separation between its images."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "twinkles_sys = 27\n",
+    "test_images = twinkles_agn_truth.query('twinkles_system == %i' % twinkles_sys) \n",
+    "test_images"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "test_lens = twinkles_lens_truth.query('galid == %i' % test_images['lens_galids'].iloc[0])\n",
+    "test_lens"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we use the WCS for the `calexp` and the ra, dec for our lensing galaxy and images to get pixel coordinates."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x_lens, y_lens = wcs.skyToPixel(SpherePoint(Angle(float(test_lens['ra']), degrees),\n",
+    "                                            Angle(float(test_lens['dec']), degrees)))\n",
+    "print(x_lens, y_lens)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x_image_1, y_image_1 = wcs.skyToPixel(SpherePoint(Angle(float(test_images['ra'].iloc[0]), degrees),\n",
+    "                                                  Angle(float(test_images['dec'].iloc[0]), degrees)))\n",
+    "x_image_2, y_image_2 = wcs.skyToPixel(SpherePoint(Angle(float(test_images['ra'].iloc[1]), degrees),\n",
+    "                                                  Angle(float(test_images['dec'].iloc[1]), degrees)))\n",
+    "print(x_image_1, y_image_1)\n",
+    "print(x_image_2, y_image_2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Display a Twinkles System"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's setup a bounding box to create a cutout of just this system."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bbox = geom.Box2I()\n",
+    "bbox.include(geom.Point2I(x_lens-150, y_lens-150))\n",
```

Comment:
It's not clear to me why this works.  I think it would be more explicit to do something like:
`bbox = geom.Box2I(geom.Point2I(x_lens-75, y_lens-75), geom.Extent2I(150, 150))`

But I guess you'd may need to edit your plotting routines below?

Reply:
This also wasn't actually doing anything. I've changed the notebook so we use a `bbox` now like you outlined here.